powerauth-data-adapter/src/main/java/io/getlime/security/powerauth/lib/dataadapter/controller/BankAccountController.java

```diff
@@ -51,9 +52,9 @@ public BankAccountController(DataAdapter dataAdapter) {
      * @return Response with user details.
      */
     @RequestMapping(value = "/list", method = RequestMethod.POST)
-    public @ResponseBody ObjectResponse<BankAccountListResponse> fetchBankAccounts(@RequestBody ObjectRequest<BankAccountListRequest> request) throws MethodArgumentNotValidException {
+    public @ResponseBody ObjectResponse<BankAccountListResponse> fetchBankAccounts(@RequestBody ObjectRequest<BankAccountListRequest> request) throws MethodArgumentNotValidException, UserNotFoundException {
         BankAccountListRequest bankAccountListRequest = request.getRequestObject();
-        BankAccountListResponse response = dataAdapter.fetchBankAccounts(bankAccountListRequest);
+        BankAccountListResponse response = new BankAccountListResponse(bankAccountListRequest.getUserId(), dataAdapter.fetchBankAccounts(bankAccountListRequest.getUserId()));
```

Comment:
This code could use a little polist... for example:

```java
String userId = bankAccountListRequest.getUserId();
// ...
```

Generally, I prefer slightly more lines of code with a clear meaning, than large method parameter chaining...